lemarche/www/tenders/urls.py

```diff
@@ -1,11 +1,13 @@
 from django.urls import path
 
-from lemarche.www.tenders.views import TenderAddView
+from lemarche.www.tenders.views import TenderAddView, TenderDetail, TenderListView
 
 
 # https://docs.djangoproject.com/en/dev/topics/http/urls/#url-namespaces-and-included-urlconfs
 app_name = "tenders"
 
 urlpatterns = [
     path("ajouter", TenderAddView.as_view(), name="add"),
+    path("liste", TenderListView.as_view(), name="list"),
```

Comment:
```suggestion
    path("", TenderListView.as_view(), name="list"),
```

on peut simplifier l'url de la sorte (cf par exemple ProfileFavoriteListView)